results/cmd/api/main.go

```diff
@@ -81,3 +81,23 @@ func (s *server) CreateTaskRun(ctx context.Context, req *pb.CreateTaskRunRequest
 	}
 	return taskrunFromClient, nil
 }
+
+// Get TaskRun from database and send it to watcher
+func (s *server) GetTaskRun(ctx context.Context, req *pb.GetTaskRunRequest) (*pb.TaskRun, error) {
+	database := s.db
+	rows, err := database.Query("SELECT taskrunlog FROM taskrun WHERE uid = " + req.GetUid())
```

Comment:
```suggestion
	rows, err := database.Query("SELECT taskrunlog FROM taskrun WHERE uid = ?", req.GetUid())
```

As is, this is susceptible to SQL injection (e.g. if req.uid = "foo; DROP TABLE taskruns")

Reply:
done